src/Surfnet/StepupRa/RaBundle/Controller/RaManagementController.php

```diff
@@ -91,9 +92,12 @@ public function raCandidateSearchAction(Request $request)
 
         $logger->notice(sprintf('Searching for RaCandidates within institution "%s"', $institution));
 
+        $pageNumber = $request->get('p', 1);
+        Assertion::digit($pageNumber, 'Expected page number to be an integer, got "%s"');
+
         $command                 = new SearchRaCandidatesCommand();
         $command->institution    = $institution;
-        $command->pageNumber     = $request->get('p', 1);
+        $command->pageNumber     = $pageNumber;
```

Comment:
As discussed in person, let's mirror how it is done in the searchAction of the [SecondFactorController](https://github.com/SURFnet/Stepup-RA/blob/45be0aebb1440d780cc01e42176bebe3aca896e8/src/Surfnet/StepupRa/RaBundle/Controller/SecondFactorController.php#L46).